src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -162,12 +205,6 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
 
     Motion *solution       = lastGoalMotion_;
 
-    // \todo Make this variable unnecessary, or at least have it
-    // persist across solve runs
-    base::Cost bestCost    = opt_->infiniteCost();
-
-    bestCost_ = opt_->infiniteCost();
-
```

Comment:
OK.
I would really like someone to double check me here, as I have tried to clean up this weird `bestCost`/`bestCost_` thing. Here both variables were set to the same thing.